api/export_pr_test.go

```diff
@@ -0,0 +1,148 @@
+package api
+
+import (
+	"bytes"
+	"encoding/json"
+	"strings"
+	"testing"
+
+	"github.com/MakeNowJust/heredoc"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+func TestIssue_ExportData(t *testing.T) {
+	tests := []struct {
+		name       string
+		fields     []string
+		inputJSON  string
+		outputJSON string
+	}{
+		{
+			name:   "simple",
```

Comment:
I think it would be nice to add two additional tests. One where a field is requested that is not in the input JSON just to verify that it does not blow up and a second where the field requested is not a valid field on an issue. Same goes for the PullRequest tests.

Reply:
Good call! I've added an assertion that it's impossible to specify invalid fields in `--json` arguments. That should take care of unsanitized values making it through to the exporter.

As for the GraphQL builder and exporters themselves, they support arbitrary values so that they are easily extensible with new fields without special-casing them. However, requesting an invalid field would likely fail on the GraphQL request phase.